Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3195,7 +3195,7 @@ void WebPageProxy::dispatchActivityStateChange()
 #if ENABLE(POINTER_LOCK)
     if (((changed & ActivityState::IsVisible) && !isViewVisible()) || ((changed & ActivityState::WindowIsActive) && !protectedPageClient()->isViewWindowActive())
         || ((changed & ActivityState::IsFocused) && !isViewFocused()))
-        requestPointerUnlock();
+        requestPointerUnlock([](bool) { });
```

Comment:
I assume all these empty lambdas should probably do the same thing as WebViewImpl::windowWillBeginSheet does, unless there is a clear reason why they don't need to.